hooks/ui_config_tk-houdini.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 Autodesk, Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+
+import sgtk
+from sgtk.platform.qt import QtGui
+
+HookClass = sgtk.get_hook_baseclass()
+
+
+class HoudiniUIConfig(HookClass):
+    """Hook to customize customize the main file view for Houdini."""
+
+    def get_item_background_color(self, item):
+        """Returns the brush to use to draw the background for this widget
+
+        :param item: The model item
+        :type item: :class:`sgkt.platofrm.qt.QtGui.QStandardItem`
+
+        :return: The QBrush.
+        :rtype: :class:`sgtk.platform.qt.QtGui.QBrush`
+        """
+
+        parent = item.index().parent()
+        if parent.isValid():
+            return QtGui.QApplication.palette().base()
+        return None
```

Comment:
Useless line. If there is no return, a Python function returns None.

But I'd prefer a fail-first strategy instead (easier to maintain IMO):
```python
parent = item.index().parent()
if not parent.isValid():
    return

return QtGui.QApplication.palette().base()
```

Not a blocker. Just my preference :)

Reply:
Returning `None` here effectively says "use the default brush". When it is time to draw, Qt will use the preferred default brush when the brush is None.

I can remove the `return None` line though.